projects/driver_controls/src/mechanical_brake_fsm.c

```diff
@@ -0,0 +1,78 @@
+#include "mechanical_brake_fsm.h"
+#include "input_event.h"
+#include "event_arbiter.h"
+#include "log.h"
+
+// Mechanical Brake FSM state definitions
+
+FSM_DECLARE_STATE(state_engaged);
+FSM_DECLARE_STATE(state_disengaged);
+
+// Mechanical Brake FSM transition table definitions
+
+FSM_STATE_TRANSITION(state_engaged) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_MECHANICAL_BRAKE, state_disengaged);
```

Comment:
Don't you need to add transitions back to itself to handle the repeated events for analog states?